message.txt

```diff
@@ -1 +1 @@
-Hi
+Hello World
```

Comment:
I changed my message to Hello World to test my mocks and stubs were working properly.  Now any message can be added to message.txt and my tests will still pass.